### PR TITLE
Adding Docker image and making minor improvement of portability

### PR DESCRIPTION
**Pull request by davidyuyuan**
There are three changes made in this PR:

1. Adding a Docker image by micromamba
2. Moving the submitter to bin/ directory
3. Making minor improvements on portability of the Python scripts

### CODE REVIEW

File: env.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+name: base
+channels:
+  - conda-forge
+dependencies:
+  - python = 3.10.2
+  - pyyaml = 6.0
+  - lxml = 4.7.1
```
Comment (nadimm-rahman) [inline env.yaml:7]: Need to add a dependency for curl, as this is used to upload the files to the Webin upload area (staging area) prior to the submission phase.
- curl = 7.71.1

**Review by nadimm-rahman — CHANGES_REQUESTED** [review]
Tested the tool using Dockerfile, and this works. I need to make some other minor changes to the scripts, but will add these after this change. Just one thing to mention, which was the addition of the curl dependency, otherwise good to go! Thank you!

File: env.yaml (hunk 1 above)
Comment (davidyuyuan) [reply env.yaml:7]: The latest version of curl is 7.81.0, slightly higher. Hope that's OK with you?

Comment (nadimm-rahman) [reply env.yaml:7]: That's fine, thank you!

**Review by nadimm-rahman — APPROVED** [review]

**Pull request merged**